examples/copy_and_pickle.py

```diff
@@ -32,10 +32,7 @@ def main():
 
     box = [(5, 5), (295, 5), (295, 295), (5, 295)]
     for i, p1 in enumerate(box):
-        if i + 1 >= len(box):
-            p2 = box[0]
-        else:
-            p2 = box[i + 1]
+        p2 = box[0] if i + 1 >= len(box) else box[i + 1]
```

Comment:
Im not a fan of the ternary operator unless in the most trivial cases (this is not one of them)